src/component-library/TextLink/TextLink.style.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+import { resolveTextColor } from '../Text';
+import { Colors } from '../utils/prop-types';
+
+type BaseTextLinkProps = {
+  $color?: Colors;
+};
+
+const BaseTextLink = styled.a<BaseTextLinkProps>`
+  color: ${({ $color }) => resolveTextColor($color)};
+
+  &[aria-disabled='true'] {
```

Comment:
Is this relevant for a text link? Never come across a disabled text link before!

Reply:
Yeah I dont think we will have to be honest. I was influenced to add here after working on CTALink, which you can disable, but thats a different thing.

Reply:
Ok, no worries :) let's take this out then